met-web/src/components/engagement/form/AddSurveyBlockTabs.tsx

```diff
@@ -43,16 +43,36 @@ const AddSurveyBlockTabs = ({
 
     // array to pass updated content to engagement context
     const surveyBlockContent = [
-        { survey_status: SubmissionStatus[SubmissionStatus.Upcoming], block_text: '' },
-        { survey_status: SubmissionStatus[SubmissionStatus.Open], block_text: '' },
-        { survey_status: SubmissionStatus[SubmissionStatus.Closed], block_text: '' },
+        {
+            survey_status: SubmissionStatus[SubmissionStatus.Upcoming],
+            block_text: savedUpcomingText ? savedUpcomingText : '',
+        },
+        {
+            survey_status: SubmissionStatus[SubmissionStatus.Open],
+            block_text: savedUpcomingText ? savedOpenText : '',
```

Comment:
isn't this savedOpenText? savedOpenText : ''

Reply:
Thanks, updated as per comments.